File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_CommandApplyPermissionsService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_CommandApplyPermissionsService : BaseService<Sys_CommandApplyPermissions, ISys_CommandApplyPermissionsRepository>
    , ISys_CommandApplyPermissionsService
    {
    private readonly ISys_CommandApplyPermissionsRepository _Sys_CommandApplyPermissionsRepository;
    public Sys_CommandApplyPermissionsService(ISys_CommandApplyPermissionsRepository Sys_CommandApplyPermissionsRepository) : base(Sys_CommandApplyPermissionsRepository)
    {
    _Sys_CommandApplyPermissionsRepository = Sys_CommandApplyPermissionsRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_CommandApplyPermissions.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_CommandApplyPermissions } from './Sys_CommandApplyPermissions.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_CommandApplyPermissionsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Command_Id', 'Function_Id', 'Menu_Id'];

    List: Sys_CommandApplyPermissions[] | undefined;
    ListFilter: Sys_CommandApplyPermissions[] | undefined;
    FormData!: Sys_CommandApplyPermissions;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_CommandApplyPermissions";
    }
}

